resources/builder.inc.sh

```diff
@@ -1269,6 +1270,7 @@ _builder_parse_expanded_parameters() {
   _builder_build_deps=--deps
   builder_verbose=
   builder_debug=
+  local builder_release=
```

Comment:
```suggestion
  local is_release=false
```

Don't use the `builder_` prefix for local variables